topics/slack/index.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Looks like the Slack files slipped into this PR as well ☝🏻

Reply:
Will change it 👍🏻